### DISCUSSION TITLE
FACING THE PROBLEM IN FLAME APPLICATION AFER USING TimewarpML PLUGIN

### DISCUSSION THREAD

**Discussion by abhiafter** [community/community#36027] [Other Feature Feedback, Questions, & Ideas]
**We are using one plugin sach as (flameTimewarpML-linux.v0.4.3) downloaded from the forum page of logik Academy.

We are facing issue like Flame Software not getting launched properly and got stucked sometimes

So please let us know if it is suitable and safe to use this plugin for FLAME 2022.3 ?**

**Comment by queenofcorgis** [comment]
Hi @abhiafter,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on logik Academy. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
